### DISCUSSION TITLE
No graphs displayed in reports, dashboard etc

### DISCUSSION THREAD

**Discussion by faheemhusain** [firefly-iii/firefly-iii#9441] [General]
I have created two asset accounts and once expense account. I have manually logged all my personal transactions for the one month divided into three categories. All the transactions are in newly created currency Saudi Riyal (SAR) (SR).

Don't worry about the privacy of listed transaction details as they are mostly not relevant but as part of POC.

Following are the transactions:
![image](https://github.com/user-attachments/assets/31621f35-1f77-4c79-92d1-ca1c102abcd4)

I can see the transactions listed under each categories:
![image](https://github.com/user-attachments/assets/1b8a85a6-a403-4068-b1f0-9ad684ccdb02)

However, dashboard looks like this:
![image](https://github.com/user-attachments/assets/dfb9bf8c-78b4-463e-bd5d-fd7c90d3ab44)

Transaction page looks like this:
![image](https://github.com/user-attachments/assets/7e63834d-95bd-48f3-b843-51e9446d0d10)

Category report with following parameters looks like this:
![image](https://github.com/user-attachments/assets/1c1cf404-94a1-4b85-9eec-ae8c1048d9b0)
![image](https://github.com/user-attachments/assets/36469f54-3de5-4739-84a1-d74e009bee48)

Troubleshooting steps attempted:
- Restarted docker. Pulled latest images of all the containers.
- Fixed minor error messages shown in docker logs(STATIC_CRON_TOKEN was not set correctly). No more error logs in docker images.

**Comment by JC5** [comment]
Probably there are errors in your browser console that will help you pinpoint the issue :).

**Comment by faheemhusain** [comment]
Thanks for the lead. Updated TRUSTED_PROXIES=** in the .env file as mentioned here https://www.reddit.com/r/selfhosted/comments/11m77a9/anyone_able_to_get_firefly_working_behind_a/.

Issue resolved. Thanks.
